src/software/python_bindings.cpp

```diff
@@ -15,10 +16,12 @@
 #include "software/geom/rectangle.h"
 #include "software/geom/vector.h"
 #include "software/world/field.h"
+#include "software/world/robot.h"
+#include "software/world/world.h"
 
 namespace py = pybind11;
 
-PYBIND11_MODULE(geometry, m)
+PYBIND11_MODULE(python_bindings, m)
```

Comment:
why did we move geomtery bindings to the top level?

Reply:
I tried putting world python bindings in a separate file and python couldn't find point anymore. I think it all needs to be in one file somehow

Reply:
okay, I won't block this on this PR, but there should be a way